packages/ui/src/components/Select.tsx

```diff
@@ -1 +1,57 @@
-export { Select, type SelectProps } from 'theme-ui';
+import { Icon, type IconType, Text } from '..';
+import {
+  type SelectProps as SelectPropsUi,
+  Select as SelectUi,
+} from 'theme-ui';
+import React from 'react';
+
+export type SelectProps = SelectPropsUi & {
+  errorIcon?: IconType;
```

Comment:
Maybe we don't need an `error` icon because we have your icons defined on the theme. Use the semantic icon `error`.

https://github.com/ttoss/ttoss/blob/main/packages/theme/src/themes/Bruttal/BruttalIcons.ts